controllers/operands/networkAddons_test.go

```diff
@@ -632,6 +639,37 @@ var _ = Describe("CNA Operand", func() {
 			}
 		}
 
+		kubevirtIpamTester := func(o ipamAnnotationParams) {
```

Comment:
please inline in the DescribeTable that uses this function. I can't see the benefit of this function, and even less benefit of assign it to a variable. I find it more readable to inline the logic in the Describe table itself.

Please also do the same for `ksdTester`

Reply:
`ksdTester` is used twice, so we can't do that there as straight forward as here
and i think any change to `ksdTester` deserves please a separate PR
it is totally orthogonal to this PR
having a topic centric PRs help later when need references and such
is it ok ?